### DISCUSSION TITLE
Add option for no auth for smtp in mail channel

### DISCUSSION THREAD

**Discussion by jobcespedes** [chatwoot/chatwoot#8141] [Feature Requests]
### Is your feature or enhancement related to a problem? Please describe.

An enhancement since some relay services and infra work without user, password authentication (IP address for instance)

### Describe the solution you'd like

When adding a mail channel and enabling smtp, an option for no auth in Authentication, and allow empty password and user, or ignore them, if that is the case

### Describe alternatives you've considered

- Use global smtp configuration, that works but only under one email address

### Additional context

N/A

**Comment by pranavrajs** [comment]
Moving this to feature requests discussion. @jobcespedes Can you please provide more context here? Do the regular support email clients have this option?

**Reply by jobcespedes** [reply]
In my knowledge, no auth in smtp is possible in several scenarios. For instance, Google Workspace allows configuring smtp relay with either smtp auth or IP address. If the latter is used, no auth is required. However, client should connect from the IP address configured. 

Regarding mail clients, Evolution allows no smtp auth. There is a check to enable smtp auth. If disabled, no credentials are used.
